changelog/5.1.3.md

```diff
@@ -0,0 +1,58 @@
+# Changelog for version 5.1.3
+
+### Released 2 December 2022
+
+## Web UI Changes
+
+### Bug Fixes
+
+* Fixed an issue where live update interval form value is ignored in the query studio.
+* Fixed an issue where macro edit page save button is incorrectly disabled.
+* Fixed an issue where table would lock browser in loop.
+* Fixed an issue with suggestion selection on label input.
+* Fixed an issue where details pane total count was wrong after changing ax.
+* Fixed an issue where some ingester configs would be labelled as 'listener' configs.
+* Fixed an issue where flows could not be duplicated by non-admins if the flow was global.
+* Prevented context submenu from being cutoff in Query Studio view.
+* Updated username value in app.comp when user is nil.
```

Comment:
what is app.comp?

Reply:
i rewrote that line to describe the bug that was fixed rather than the fix itself.